src/streamsync/__init__.py

```diff
@@ -1,8 +1,9 @@
+import importlib.metadata
 from typing import Union, Optional, Dict, Any
 from streamsync.core import Readable, FileWrapper, BytesWrapper, Config
 from streamsync.core import initial_state, component_manager, session_manager, session_verifier
 
-VERSION = "0.3.0"
+VERSION = importlib.metadata.version("streamsync")
```

Comment:
So the idea is to switch to defining version in the package and then fetching it from the code? Makes more sense I guess

Reply:
That's the recommended way to manage this situation. Another one is to have a utility as bumpversion to update both location. I would avoid this one.

Reply:
I really like this option.